src/pages/games/[id].tsx

```diff
@@ -88,13 +90,24 @@ const GamePage = () => {
         </>
       )}
       <EmbeddedGame gameId={gameId as string} />
-      <TabsComponent mode="view" gameData={gameData} />
+      <TabsComponent
+        mode="view"
+        gameData={gameData}
+        admin={userData && userData.label === "administrator"}
```

Comment:
maybe create a const to check if the user can see this tab (parent, educator, administrator) also line 105